js/assessments/checkForTooLongSentences.js

```diff
@@ -0,0 +1,17 @@
+var forEach = require( "lodash/forEach" );
+
+/**
+ * Counts number of too long sentences.
+ * @param {array} sentences The array containing sentences.
+ * @returns {number} Number of too long sentences.
+ */
+module.exports = function ( sentences ) {
+	var tooLong = 0;
+	var recommendedValue = 20;
+	forEach( sentences, function( sentence ) {
```

Comment:
This can also be:
`var tooLong = sentences.filter( function( sentence ) { return sentence > recommendedValue } );`
`return tooLong.length;`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter